pom.xml

```diff
@@ -312,6 +314,16 @@
                 <type>test-jar</type>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>io.gravitee.cockpit</groupId>
+                <artifactId>gravitee-cockpit-api</artifactId>
+                <version>${gravitee-cockpit-api.version}</version>
+            </dependency>
+            <dependency>
```

Comment:
why do we need this dependency ?

Reply:
This dependency is mandatory because it contains all XXXCommand objects we need to handle to communicate with Cockpit.